Source/Mapping.AutoMapper/ITwoWayMappingProfile.cs

```diff
@@ -1,7 +1,9 @@
-﻿using AutoMapper;
+﻿using System;
+using AutoMapper;
 
 namespace Affecto.Mapping.AutoMapper
 {
+    [Obsolete]
```

Comment:
Should we add a comment, too? [ObsoleteAttribute("Two-way mapping is deprecated and it is going to be removed in a future major version.", false)]

Reply:
Sorry, I meant [Obsolete("Two-way mapping is deprecated and it is going to be removed in a future major version.", false)]